packages/ng/user/src/lib/select/input/user-select-input.module.ts

```diff
@@ -17,7 +17,7 @@ import { luUserSelectInputTranslations } from './user-select-input.translate';
 		LuUserDisplayModule,
 		LuOptionOperatorModule,
 		LuOptionPickerModule,
-		LuInputClearerModule,
+		LuInputClearerComponent,
```

Comment:
This module could rely on components instead of modules (for other imports, can't target them all in the review)